src/Faker/Provider/ps_AF/Internet.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Faker\Provider\ps_AF;
+
+class Internet extends \Faker\Provider\Internet
+{
+    protected static $userNameFormats = array(
+        '{{lastNameAscii}}.{{firstNameAscii}}',
+        '{{firstNameAscii}}.{{lastNameAscii}}',
+        '{{firstNameAscii}}##',
+        '?{{lastNameAscii}}',
+    );
+
+    protected static $safeEmailTld = array(
+        'com', 'af', 'me', 'net', 'org','co.af'
+    );
+
+    /**
+     * @link https://en.wikipedia.org/wiki/.af
+     */
+    protected static $tld = array(
+        'biz', 'com', 'info', 'af', 'net',
+    );
+
+    /**
+     * @link https://en.wikipedia.org/wiki/Category:Pashto-language_surnames
+     */
+    protected static $lastNameAscii = array(
+        'momand', 'shinwari', 'abdali', 'safi', 'adamkhel', 'afridi', 'akakhel', 'alikhel', 'allaiwal', 'babai',
+        'baizais', 'bangash', 'banoori', 'banuchi', 'barozai', 'bazai', 'bettani', 'changezi', 'dawar',
+        'dawi', 'deshiwal', 'dirani', 'durrani', 'gandapur', 'gharghashti', 'ghilji',
+        'gul', 'gulwal', 'hafizkhel', 'harifal', 'ibrahimkhel', 'ibrahimzai', 'ibrahimzai',
+        'jadun', 'jalalzai', 'kakar', 'kalakhel', 'kharotakhel', 'khudiadadzai', 'lodi', 'lohani', 'mandokhel', 'miani',
+        'mirdadzai', 'musakhel', 'nasozai', 'nattuzai', 'niazi', 'painda', 'panni', 'sarwani', 'shirani', 'sur',
+        'tikriwal', 'turkhel','yaqubzai',
+    );
+
+    /*
+     * @link https://www.behindthename.com/submit/names/usage/pashto
```

Comment:
There is copyright on the source data, so please provide a valid open source

Reply:
@pimjansen Can you please check now?